pos_receipt_replace_user_by_trigram/views/res_config_settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record
+        id="pos_receipt_hide_user_res_config_settings_view_form_inherit"
+        model="ir.ui.view"
+    >
+        <field
+            name="name"
+        >pos.receipt.hide.user.res.config.settings.view.form.inherit</field>
```

Comment:
```suggestion
```

💡 tip: odoo automatically generates the view's name

unless you want to give a meaningful name, you can ignore this :)